mitmproxy/utils/typecheck.py

```diff
@@ -1,20 +1,37 @@
 import typing
 
 
-def check_type(name: str, value: typing.Any, typeinfo: typing.Any) -> None:
+def check_command_return_type(value: typing.Any, typeinfo: typing.Any) -> bool:
```

Comment:
We support a different restricted set of types for each of command arguments, command return values and options. They may converge later, but I'm carefully expanding the types one piece at a time at the moment.